### DISCUSSION TITLE
Exporting model after training with continual learning

### DISCUSSION THREAD

**Discussion by Sean3216** [ContinualAI/avalanche#1278] [Q&A] [answered]
Hi! I'm new to Continual Learning and I found the package to be very great!
I have a question about how to export a model after training with Continual Learning.

I want to train a model with continual learning and then export the model to be used later on. **Is this possible in Avalanche?** I have tried to search the documentation but I haven't found the answer.

Cheers!

**Comment by AntonioCarta** [comment, marked as answer]
Hi, you can save `strategy.model`. It's a PyTorch `nn.Module`.

**Reply by Sean3216** [reply]
Thank you for the brief answer. Cheers!
